### PR TITLE
Job filter style fix

### PR DESCRIPTION
**Pull request by mtrna**
before:
before

after:
after

### CODE REVIEW

File: azkaban-web-server/src/main/less/base.less [hunk 1]
```diff
@@ -7,8 +7,8 @@
 }
 
 .container-fill {
-  position: absolute;
-  top: 230px;
+  position: relative;
```
Comment (kunkun-tang) [inline azkaban-web-server/src/main/less/base.less]: This tag looks being used by many other pages, like flowpage, jobdetailpage. Have you checked that everywhere is fine?

Comment (mtrna) [reply azkaban-web-server/src/main/less/base.less]: Thanks for pointing it out; I have checked the other tabs an there was impact.

Moved all the styling logic to `div#graphView.container-fill`, which is specific to the graph pane, and tested the updated version on chrome and firefox, it looks good to me now.

**Comment by HappyRay** [comment]
@mtrna Are you no longer interested in finishing this PR? Is there anything we can do to help?

**Comment by mtrna** [comment]
Hi @HappyRay ! I am interested, the MR is reopened here: https://github.com/azkaban/azkaban/pull/1685

**Pull request closed without merging**